.github/workflows/build.yml

```diff
@@ -82,18 +82,6 @@ jobs:
           --health-interval 1s
           --health-timeout 5s
           --health-retries 60
-      consul:
```

Comment:
Probably want to revert this since the build is running in a docker container.

Reply:
Thanks for explaining, fixed